### PR TITLE
pkg/alertmanager: minor code clean-up

### PR DESCRIPTION
**Pull request by simonpasquier**
## Description

Use constants instead of hardcoded strings.

## Type of change

_What type of changes does your code introduce to the Prometheus operator? Put an `x` in the box that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [X] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Changelog entry

_Please put a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note

```

### CODE REVIEW

File: pkg/alertmanager/statefulset.go [hunk 1]
```diff
@@ -35,21 +35,30 @@ import (
...
+	alertmanagerConfigOutDir           = "/etc/alertmanager/config_out"
+	alertmanagerConfigFile             = "alertmanager.yaml"
+	alertmanagerConfigFileCompressed   = "alertmanager.yaml.gz"
+	alertmanagerConfigEnvsubstFilename = "alertmanager.env.yaml"
+
+	alertmanagerStorageDir = "/alertmanager"
+
+	sSetInputHashName = "prometheus-operator-input-hash"
+
+	defaultPortName = "web"
```
Comment (PhilipGough) [inline pkg/alertmanager/statefulset.go:61]: should we move this up with the other `defaultRetention `

File: pkg/alertmanager/statefulset.go [hunk 2]
```diff
@@ -35,21 +35,30 @@ import (
...
-	sSetInputHashName                    = "prometheus-operator-input-hash"
-	defaultPortName                      = "web"
+	governingServiceName = "alertmanager-operated"
+
+	defaultRetention = "120h"
+
+	tlsAssetsVolumeName                = "tls-assets"
+	tlsAssetsDir                       = "/etc/alertmanager/certs"
+	secretsDir                         = "/etc/alertmanager/secrets/"
+	configmapsDir                      = "/etc/alertmanager/configmaps/"
```
Comment (PhilipGough) [inline pkg/alertmanager/statefulset.go:45]: we can strip trailing slash here now since using path.Join if its preferred

**Review by PhilipGough — APPROVED** [review]
lgtm

thanks - two nits/comments but feel free to ignore

**Comment by simonpasquier** [comment]
@PhilipGough thanks! I've applied your suggestions.

**Review by JoaoBraveCoding — APPROVED** [review]
lgtm, CI failure seems like a flake

**Pull request merged**